packages/eslint-plugin/src/configs/recommended-hulu.ts

```diff
@@ -6,5 +6,6 @@ export = {
         "@hulu/brightscript/onkeyevent-boolean-return-type": "error",
         "@hulu/brightscript/no-unsafe-conditions": "error",
         "@hulu/brightscript/no-typed-func-params": "warn",
+        "@hulu/brightscript/comment-style": ["error", "'"],
```

Comment:
I forgot that `recommended-hulu` doesn't extend `recommended`.  Perhaps it should 🤷